modules/apps/sharing/sharing-service/src/main/java/com/liferay/sharing/internal/search/spi/model/index/contributor/SharingEntryModelDocumentContributor.java

```diff
@@ -59,6 +65,30 @@ private Map<Locale, String> _getTitleMap(SharingEntry sharingEntry) {
 		return sharingEntryInterpreter.getTitleMap(sharingEntry);
 	}
 
+	private boolean _isCms(long groupId) {
+		try {
+			Group group = _groupLocalService.getGroup(groupId);
+
+			return group.isDepot();
+		}
+		catch (PortalException portalException) {
+			if (_log.isWarnEnabled()) {
+				_log.warn(
+					"Unable to get group " + groupId +
+						" while indexing document",
+					portalException);
+			}
+		}
+
+		return false;
```

Comment:
I just noticed this now, sorry: why not do a `fetch*` instead of a get? It looks like the only exception throws is a `NoSuch*`. The code would look simpler by using fetch + nullness check:
```
Group group = _groupLocalService.fetchGroup(groupId);

if (group == null) {
    return false;
}

return group.isDepot();
```

Reply:
I was thinking more on throwing the log to let the system to know that is because the group was wrong. 
But as you prefer, no strong opinion :)

Reply:
and that message can be logged on the == null
you ok with the _log or is not compilant with the SF or Brian?